lighthouse-core/test/audits/byte-efficiency/uses-responsive-images-test.js

```diff
@@ -120,80 +122,84 @@ describe('Page uses responsive images', () => {
           null
         ),
       ],
-    });
+    },
+      []
+    );
 
     assert.equal(auditResult.items.length, 0);
   });
 
   it('identifies when images are not wasteful', () => {
+    const networkRecords = [generateRecord(100, 300, 'https://google.com/logo.png'), generateRecord(90, 500, 'https://google.com/logo2.png'), generateRecord(20, 100, 'data:image/jpeg;base64,foobar')];
     const auditResult = UsesResponsiveImagesAudit.audit_({
       ViewportDimensions: {innerWidth: 1000, innerHeight: 1000, devicePixelRatio: 2},
       ImageElements: [
         generateImage(
           generateSize(200, 200),
           generateSize(450, 450, 'natural'),
-          generateRecord(100, 300),
           'https://google.com/logo.png'
         ),
         generateImage(
           generateSize(100, 100),
           generateSize(210, 210, 'natural'),
-          generateRecord(90, 500),
           'https://google.com/logo2.png'
         ),
         generateImage(
           generateSize(100, 100),
           generateSize(80, 80, 'natural'),
-          generateRecord(20, 100),
           'data:image/jpeg;base64,foobar'
         ),
       ],
-    });
+    },
+      networkRecords
+    );
 
     assert.equal(auditResult.items.length, 2);
   });
 
   it('ignores vectors', () => {
     const urlA = 'https://google.com/logo.svg';
     const naturalSizeA = generateSize(450, 450, 'natural');
-    const recordA = generateRecord(100, 300, 'image/svg+xml');
-
+    // eslint-disable-next-line max-len
+    const image = {...generateImage(generateSize(10, 10), naturalSizeA, urlA), mimeType: 'image/svg+xml'};
```

Comment:
```suggestion
    const image =
      {...generateImage(generateSize(10, 10), naturalSizeA, urlA), mimeType: 'image/svg+xml'};
```

another option (indentation might be off in github UI but the intent is to start `{` two spaces after column `=` is on.)